### DISCUSSION TITLE
Hidden Text? (Almost...)

### DISCUSSION THREAD

**Discussion by Legsmaniac** [Jyers/Marlin#114] [Q&A] [answered]
Firstly, thanks for a great Marlin fork.  Lovin' it so far.

I'm using the latest (as of last night) Extensible UI v1.1.1 fork, self compiled.

Any ideas why there's what looks like some almost hidden text (possibly Chinese?) behind the three locations on the LCD where text information is displayed?  There seems to be a long black rectangular box not quite reaching the edges of the screen where information text appears but at the very ends of this black box seems to be some text underneath the box, can't make my mind up whether it's Chinese characters or not but text nonetheless.  See where I have circled in photo below.....

![HiddenText](https://user-images.githubusercontent.com/50076851/108845076-e165e500-75d4-11eb-9828-8492e0d02655.jpg)

Is this a bug?  Or something I have done wrong in the config options?
Thank you.

**Comment by Jyers** [comment]
That is very strange. I have yet to experience such an issue, but I will see if I can find a way to recreate it and hopefully find a fix.

**Comment by Legsmaniac** [comment]
Apologies.  I meant to reply and post here again to update.....

After turning off the printer and later turning on again, the above weird effects have not presented itself again - so far.
Everything is OK, everything looks correct and normal.
Have no idea how or why I had that weird anomaly.
I'll let you know if it happens again.

**Comment by feelfree69** [comment]
I see that here too, probably an artifact due to the very long (and scrolling) filename.

**Comment by Jyers** [comment, marked as answer]
If you continue to see this issue, feel free to post a GitHub issue about it, perhaps with some detailed steps to recreate it so I can find the source of the problem.

**Comment by Legsmaniac** [comment]
Thanks. Sorry but so far it's not presented itself again yet. Not that I've done much printing recently owing to not feeling too well plus I'm waiting on some more supplies for next projects. Should be here by the weekend or Monday latest so we'll see what happens next week. Long file names are generated automatically by Prusa slicer so I'm sure there'll be some more to test with soon. Saying that, I've since grabbed and compiled the newest firmware as of a couple days ago hence the filament runout error report I made recently. I have yet to grab the latest with the fix but I will soon so thanks for that. 👍
